app/app.go

```diff
@@ -110,6 +113,9 @@ import (
 	"github.com/umee-network/umee/v2/x/oracle"
 	oraclekeeper "github.com/umee-network/umee/v2/x/oracle/keeper"
 	oracletypes "github.com/umee-network/umee/v2/x/oracle/types"
+
+	// unnamed import of statik for swagger UI support
+	_ "github.com/umee-network/umee/v2/client/docs/statik"
```

Comment:
Should this be moved to tools/tools.go?

Reply:
If we move it, than the swagger will use the cosmos-sdk statik file which will display the swagger for cosmos-sdk API